CHANGELOG

```diff
@@ -1,3 +1,11 @@
+## [0.5.0] - 2023-08-12
+### Breaking Changes
+### New Features
+- Use CentOS stream 9 on traffic-gen and VM under test images
```

Comment:
We wrote the base image version in previous releases.

Reply:
ah, then removing

Reply:
I don't think this line should have been removed.
I referred to the fact we wrote the base image version, for example:https://github.com/kiagnose/kubevirt-dpdk-checkup/blob/d67d80a9664df6c822e9dfac6eabfd1ef7c882b8/CHANGELOG#L15

Reply:
DONE